docs/api/app.md

```diff
@@ -1307,6 +1307,76 @@ Sets the application's [dock menu][dock-menu].
 
 Sets the `image` associated with this dock icon.
 
+### `app.sysInfo.numberOfProcessors()`
```

Comment:
@miniak I know you're just following the Chromium names, but I think we could make these more Node/Electron-ish. For exampe, `processorCount()`, `totalPhysicalMemory()`, `availablePhysicalMemory()`, `freeDiskSpace()`, etc. WDYT?